tools/cwtool/TxSimulator.hs

```diff
@@ -155,7 +155,7 @@ simulate sc@(SimConfig dbDir txIdx' _ _ cid ver gasLog doTypecheck) = do
           withRocksDb "txsim-rocksdb" modernDefaultOptions $ \rdb ->
             withBlockHeaderDb rdb ver cid $ \bdb -> do
               let pse = PactServiceEnv Nothing cpe paydb bdb getGasModel readRewards 100 0 ferr
-                        ver True False logger gasLogger (pactLoggers cwLogger) False 1 defaultBlockGasLimit cid
+                        ver False logger gasLogger (pactLoggers cwLogger) False 1 defaultBlockGasLimit cid
```

Comment:
Unrelated to this PR, but I think, it would be nicer (and easier to read and maintain) if verbose record initialization syntax would be used here.

Reply:
Totally agree.